src/components/file-list/file-title/file-title.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import propTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+
+import { getFileNameWithoutExtension } from './file-title.utils';
+
+export const FileTitle = ({ title }) => {
+  const isExtensionOn = useSelector((state) =>
+    state.settings.get('fullFilename'),
+  );
+  const formattedTitle = isExtensionOn
+    ? title
+    : getFileNameWithoutExtension(title);
```

Comment:
With the new implementation of this function, this ternary is no longer required.

Reply:
It is necessary because the user can toggle if whether they want to see the extension or not.